packages/backend/src/index.ts

```diff
@@ -3,6 +3,8 @@ import { createBackendApiClient } from './api';
 import type { CreateAuthenticateRequestOptions } from './tokens';
 import { createAuthenticateRequest } from './tokens';
 
+export type { InstanceKeys } from './tokens';
```

Comment:
Good job adding `type` here.
Question: is this now a public export of the package? If so, do we want it to be?

Reply:
@nikosdouvlis yes this is a public export now and it is used in `sdk-node` package.